__tests__/integrations.test.js

```diff
@@ -347,6 +349,26 @@ describe('Articles', () => {
         });
     });
   });
-  
-  
 });
+
+// USERS
+describe('Users', () => {
+  describe('USERS: /api/users', () => {
+    test('GET 200: Endpoint returns all users', () => {
+      return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({ body: { users } }) => {
+          const expectedObject = {
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          };
+          users.forEach((user) => {
+            expect(Object.keys(user).length).toBe(3);
```

Comment:
What if we were to add a new property to our users in the future? Similar to how we use .toMatchObject or a similar flexible jest matcher we want to avoid brittle tests that may fail in the future due to changes.